src/status_im2/contexts/quo_preview/navigation/bottom_nav_tab.cljs

```diff
@@ -36,7 +37,15 @@
                :value :unread-dot}]}
    {:label "Counter Label"
     :key   :counter-label
-    :type  :text}])
+    :type  :text}
+
+   {:label   "Customization color"
+    :key     :customization-color
+    :type    :select
+    :options (map (fn [[k _]]
```

Comment:
do we need the `_` argument there?

Reply:
```clj (map (fn [k] ...) (keys colors/customization)```

Reply:
@J-Son89 Yes, We need it, even if ClojureScript is more permissive, I'd add it as a good practice.

@ibrkhalil Thanks for the suggestion, in fact I already thought about that option but I didn't take it because it requires to travel 2 times the collection instead of one (at cost of a destructuring), I mean:

`(keys colors/customization)` creates a new seq based on a map, and
`(map (fn [k] ,,,)` creates a new seq based on the previous seq

Instead `(map (fn [k _],,,) colors/customization)` is a single pass on the seq to create it.

I understand this is a very small optimization, I just didn't feel confortable traveling twice when adding a destructuring avoids it.

Reply:
Agree that your solution is more performant, Amazing job!
I like the optimize where I can mindset, Keep it up.